x/wasm/client/rest/gov.go

```diff
@@ -407,7 +407,50 @@ func (s UnpinCodeJSONReq) GetBaseReq() rest.BaseReq {
 
 func UnpinCodeProposalHandler(cliCtx client.Context) govrest.ProposalRESTHandler {
 	return govrest.ProposalRESTHandler{
-		SubRoute: "pin_code",
+		SubRoute: "unpin_code",
+		Handler: func(w http.ResponseWriter, r *http.Request) {
+			var req UnpinCodeJSONReq
+			if !rest.ReadRESTReq(w, r, cliCtx.LegacyAmino, &req) {
+				return
+			}
+			toStdTxResponse(cliCtx, w, req)
+		},
+	}
+}
+
+type UpdateInstantiateConfigProposalJSONReq struct {
+	BaseReq rest.BaseReq `json:"base_req" yaml:"base_req"`
+
+	Title       string    `json:"title" yaml:"title"`
+	Description string    `json:"description" yaml:"description"`
+	Proposer    string    `json:"proposer" yaml:"proposer"`
+	Deposit     sdk.Coins `json:"deposit" yaml:"deposit"`
+
+	CodeUpdates []types.CodeAccessConfigUpdate `json:"code_updates" yaml:"code_updates"`
```

Comment:
this follows the protobuf defitinion 
```
repeated CodeAccessConfigUpdate code_updates = 3
```

should we rename the proto field to code_access_config_updates ? 

I think it would make sense because is just doing an specific action and not performing a full update to a code id.

thoughts?

Reply:
I think `access_config_updates` or `code_access_config_updates` would work (and ideally update both in the Protobuf file as well)